ReactWindows/ReactNative.Shared/Views/View/ReactViewManager.cs

```diff
@@ -30,10 +31,57 @@ private enum Radius
             BottomRight,
         }
 
+        private class BorderProps
+        {
+            public uint? Color;
+        }
+
+        private readonly ConditionalWeakTable<BorderedCanvas, BorderProps> _borderProps =
```

Comment:
>ConditionalWeakTable [](start = 25, length = 20)

Any reason to use CWT instead of dictionary here? There's a lifecycle event on `IViewManager` (`OnDropViewInstance` or something of the sort) that gives you a last chance to clean up entries in the dictionary.

Reply:
IIRC CWT is not nearly as performant, although it does have that nice `GetOrCreate` functionality...

---
In reply to: [154768024](https://github.com/Microsoft/react-native-windows/pull/1521#discussion_r154768024) [](ancestors = 154768024)

Reply:
I found that BaseViewManager overrides OnDropViewInstance and resorted to CWT.